coops-ui/src/components/AnnualReport/Directors.vue

```diff
@@ -712,18 +715,23 @@ export default class Directors extends Mixins(DateMixin, ExternalMixin, EntityFi
   }
 
   /**
-   * Computed value.
-   * @returns The latest of the most recent COD filing and the last pre-load Cobrs filing.
-   */
+     * Computed value.
+     * Determine the latest of the following dates:
+     * - the last COD filing in filing history (from legal DB)
+     * - the last AR filing in filing history (from the Legal DB)
+     *
+     * If the entity has no filing history, the founding date will be used.
+     */
   private get earliestDateToSet (): string {
     let earliestDateToSet = null
 
-    if (this.lastCODFilingDate === null) {
-      earliestDateToSet = this.lastPreLoadFilingDate
-    } else if (this.compareDates(this.lastCODFilingDate, this.lastPreLoadFilingDate, '>')) {
-      earliestDateToSet = this.lastCODFilingDate
+    if (this.lastCODFilingDate === null && this.lastAnnualReportDate == null) {
```

Comment:
Just curious why some comparisons are using `==` to type check and others are explicitly checking `===` for `null`. Is it to purposely check to check for `null` OR `undefined` in the instance of `this.lastAnnualReportDate` ?

Reply:
No no. I think I missed it . Will fix it . Thanks :)

Reply:
Np at all, looks good!